types/storybook__react/storybook__react-tests.tsx

```diff
@@ -1,17 +1,31 @@
 import * as React from 'react';
-import { storiesOf, setAddon, addDecorator, addParameters, configure, getStorybook, RenderFunction, Story, forceReRender, DecoratorParameters, clearDecorators } from '@storybook/react';
+import {
+    storiesOf,
+    setAddon,
+    addDecorator,
+    addParameters,
+    configure,
+    getStorybook,
+    RenderFunction,
+    Story,
+    forceReRender,
+    load,
+    DecoratorParameters,
+    clearDecorators,
+} from '@storybook/react';
 
 const Decorator = (story: RenderFunction) => <div>{story()}</div>;
 const parameters: DecoratorParameters = { parameter: 'foo' };
 
 forceReRender();
+load(require.context('../src', true, /\.stories\.tsx$/), module);
```

Comment:
this looks good to me, however this function is only available in a technical preview right now if I understand the doc correctly that you linked @ElliottRoche?.
I can see it being used here: https://github.com/storybookjs/storybook/blob/454fab82f8be5781a93566f26070596a98bdc9c8/examples/cra-kitchen-sink/.storybook/config.js#L25 so I suspect it is released as some version, but unfortunately, I have no experience on how to publish types for a preview version, do you know?

Reply:
Yeah this for the alpha version of storybook, currently v5.2.0-alpha.38, that introduces Storybook docs as a replacement or enhancement for a lot of the addon packages like addon-info.

No idea how to publish types for a preview, but this load function didn't appear to affect anything from previous storybook versions so I figured I'd go ahead and add it.